vutil/vxs.xs

```diff
@@ -4,13 +4,7 @@
 #include "XSUB.h"
 #define NEED_sv_2pv_nolen_GLOBAL
 /* for vutil.c */
-#define NEED_my_snprintf
-#define NEED_newRV_noinc
-#define NEED_vnewSVpvf
-#define NEED_newSVpvn_flags_GLOBAL
-#define NEED_warner
 /* end vutil.c */
-#include "ppport.h"
```

Comment:
this is not needed as ppport.h is going to be added from vutil.h

Reply:
First of all, that really ought to be a separate commit.

Second of all, why would you want that in the first place? What's the advantage of moving it into vutil.h? I would make more sense to me to eliminate the `#include "ppport.h"` there, given that it has to be guarded in an `#ifndef PERL_CORE`

Reply:
#19 is splitting the `ppport.h` setup as its own PR
let's move the discussion there